scripts/await-contracts-docker.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
```

Comment:
This file is now renamed to `scripts/patch-config.js`. So this one must be pushed by a mistake.